README.adoc

```diff
@@ -111,24 +109,23 @@ The TCP system tests run only if Python selectors is available.
 pip3 install --user quart h2 selectors
 ----
 
-Websocket system tests use the Python websockets asyncio module.
+Websocket system tests use the Python `websockets asyncio` module.
```

Comment:
Name of the module is `websockets`, https://pypi.org/project/websockets/. The module does use asyncio API but that is not part of it's name.